server/controllers/apply/applicationsController.ts

```diff
@@ -42,10 +42,16 @@ export default class ApplicationsController {
 
       if (application.status !== 'inProgress') {
         const referrer = req.headers.referer
-        res.render('applications/show', { application, referrer })
-      } else {
-        res.render('applications/tasklist', { application, taskList, errorSummary, errors })
+
+        if (req.query.tab === 'notes') {
```

Comment:
I'm not sure, I agree it's a bit confusing: the page's heading is 'Notes' but it currently solely contains a timeline. 
I think we should keep it the same as the heading for now and when the ['Add a note to the application'](https://www.figma.com/file/kP7jyX3L01CQap6ErAX648/Approved-Premises-Beta?type=design&node-id=2507%3A69291&mode=design&t=BHIO4PJH1fcQ3aze-1) component is added it will make more sense?

Reply:
The up to date designs here https://www.figma.com/file/kP7jyX3L01CQap6ErAX648/Approved-Premises-Beta?type=design&node-id=3791-72313&mode=design&t=lHOigKm09s4aEqCH-4 have the title as "Timeline"

Reply:
:o  
Ok cool, I'll update - Timeline is clearer